.golangci.yml

```diff
@@ -182,10 +178,8 @@ issues:
   # changes in PRs and avoid nitpicking.
   exclude-use-default: false
   exclude-rules:
-  # Specific exclude rules for deprecated fields that are still part of the codebase. These
+  # Specific exclude rules for deprecated items that are still part of the codebase. These
   # should be removed as the referenced deprecated item is removed from the project.
```

Comment:
Just to get rid of this duplicate comment and avoid having to write 2-3 different comments for separate blocks for no good reason :)